src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -195,6 +195,24 @@ public void TestSlice()
             Assert.AreEqual("efg", StandardFilters.Slice("abcdefg", 4, 30));
             Assert.AreEqual("a", StandardFilters.Slice("abc", -4, 2));
             Assert.AreEqual("", StandardFilters.Slice("abcdefg", -10, 1));
+
+            // Test for arrays
+            var testArray = new string[] { "a", "b", "c", "d" };

```

Comment:
this should probably be moved to another method. don't forget to create a test method for other types of arrays